src/databricks/labs/ucx/source_code/notebooks/cells.py

```diff
@@ -195,6 +187,19 @@ def build_dependency_graph(self, parent: DependencyGraph) -> list[DependencyProb
         )
         return [problem]
 
+    def read_notebook_path(self):
```

Comment:
```suggestion
    def maybe_notebook_path() -> Path | None:
      ... = self._read_notebook_path()
      
    def _read_notebook_path(self):
```

return type interface exposes too much unnecessary private information, make this method private and wrap it with public method with only what we need.

```
def _load_source_from_run_cell(self, run_cell: RunCell):
        path, _, _ = run_cell.read_notebook_path()
```

Reply:
done